pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -1,8 +1,13 @@
 ---
+<%
+  title = hoc_s(:meta_tag_og_title)
+  title = I18n.t(:og_title_soon) if DCDO.get('soon-hoc', false)
+  title = I18n.t(:og_title_here) if DCDO.get('actual-hoc', false)
```

Comment:
Is it ok to mix & match `hoc_s` and `I18n.t` here?  What's the reason for having two different pathways?

Reply:
I think that it's fine - as far as I know.

Reply:
`hoc_s` was added for Tutorial Explorer (https://github.com/code-dot-org/code-dot-org/pull/11714) so that we could reuse the same strings on /learn between all of the pegasus domains.  Looks like it's become more widely used since then.  It makes sense if it's a string that's shared between hourofcode.com and code.org/csedweek.org.

Reply:
Now I'm wondering whether it's necessary to use it on pages that are exclusively on hourofcode.com.

Reply:
@tanyaparker tells me that `hoc_s` pulls from the right place even when pages are exclusively on hourofcode.com.  It makes for a simpler translation pipeline.  cc @Hamms